projects/packages/forms/.phpcs.dir.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<ruleset>
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<property name="text_domain" type="array">
+				<element value="jetpack-forms" />
```

Comment:
Does this mean we start translations from scratch? Should we ask if there's a way to "migrate" translations? Or simply copy the `jetpack` translations domain, then unused entries will start to be removed over time?

Reply:
The textdomain change should be automatically handled for us when the package will be required inside the Jetpack plugin.